File: src/test/java/com/dotnomi/fabricdependencyinjection/testclasses/postconstruct/parameter/ParameterPostConstructService.java
package com.dotnomi.fabricdependencyinjection.testclasses.postconstruct.parameter;

import com.dotnomi.fabricdependencyinjection.annotation.ModScoped;
import com.dotnomi.fabricdependencyinjection.annotation.PostConstruct;

@ModScoped
@SuppressWarnings("unused")
public class ParameterPostConstructService {
  @PostConstruct
  private void initialize(String var1) {}
}

File: src/test/java/com/dotnomi/fabricdependencyinjection/kotlin/testclasses/postconstruct/parameter/ParameterPostConstructService.kt
package com.dotnomi.fabricdependencyinjection.kotlin.testclasses.postconstruct.parameter

import com.dotnomi.fabricdependencyinjection.annotation.ModScoped
import com.dotnomi.fabricdependencyinjection.annotation.PostConstruct

@ModScoped
class ParameterPostConstructService {
    @PostConstruct
    private fun initialize(var1: String) {
    }
}
